### ISSUE TITLE
Icon request: icon-feedly

### ISSUE THREAD

**Issue by DVDCano** [FortAwesome/Font-Awesome#2086]
Icon for Feedly web service (category Brand Icons)

![feedly](https://f.cloud.github.com/assets/5596657/1255535/9aa684f8-2b86-11e3-8d3b-093dfb1bc30c.png)

![feedly-logo1](https://f.cloud.github.com/assets/5596657/1255558/04584a76-2b87-11e3-8bb9-6f0aa7d8e713.png)

http://en.gravatar.com/devhd#pic-0

Thanks!

**Comment by mariekreativ** [comment]
+1

**Comment by berendt** [comment]
+1 and please also add `icon-feedly-sqaure`.

**Comment by skruhela** [comment]
+1, Please add feedly icon

**Comment by lindsayrose** [comment]
+1 I would love this

**Comment by YuheiNakasaka** [comment]
:+1:

**Comment by quickonlinetips** [comment]
+1 yes please. needed.

**Comment by alisonmf** [comment]
+1

**Comment by MichaelTunnell** [comment]
What's the point of this? Feedly is just a proprietary RSS reader.

We already have http://fontawesome.io/icon/rss and http://fontawesome.io/icon/rss-square so what exactly would benefit the project to include a brand that essentially means the same thing?

**Comment by gaplauche** [comment]
Same reason podcasters will have an iTunes podcast icon and a generic rss icon, the first linking to their podcast on iTunes and the second for non-iTunes users.

We can use a Feedly icon for a "subscribe via Feedly" link that makes it easier for Feedly users to do so.

**Comment by MichaelTunnell** [comment]
iTunes provides a reason to use in the sense that Apple users are fairly locked down to that terrible service and doesn't really support RSS but Feedly provides no real benefit when you can just add a bookmarklet as a Feedly user or many other methods of easily adding any RSS feed to a feedly account. I guess some might want it regardless but doesn't sound very useful to me.

**Comment by cookieandkate** [comment]
Micheal - Feedly is one of the most popular RSS readers for fashion and food blogs (it's up there with Bloglovin', which I link to next to my RSS icon using your heart icon). I'm trying to grow my subscriber base and want to make it as easy as possible for people to subscribe. That's why I would really like a Feedly icon. I doubt that many Feedly users have installed the bookmarklet.

**Comment by MichaelTunnell** [comment]
@cookieandkate yes but what relevance is there to use the Feedly icon instead of the actual RSS feed icon? RSS feed icon is important to have but Feedly is not as relevant as RSS itself or as useful. http://fontawesome.io/icon/rss-square/

**Comment by gaplauche** [comment]
@MichaelTunnell I'm not sure why you're fighting this so hard. It's just one little icon. How hard could it be to include? Clearly quite a few people want it. The popularity of the service and its analog to iTunes Podcasts should be enough.

**Comment by MichaelTunnell** [comment]
I am not fighting I am asking for the reason people want this. It seems to me that the reason for this is because of the exact same reason that RSS icon exists so I don't understand the benefit of the redundancy. It isn't about how hard it is but it is about the weight of the project being loaded. To me it seems redundant and thus the weight should be used for something else.

**Comment by ReveWeber** [comment]
Linking directly to Feedly is about lowering the barrier to entry - like having a mailto: instead of just an email address. I don't think the RSS icon would be seen as synonymous by nontechnical users, who may have experienced clicking RSS links and getting a pageful of XML for their trouble.

+1 to this from me.

**Comment by Alvarz** [comment]
+1

**Comment by FiliMX** [comment]
+1

**Comment by aethior** [comment]
+1

**Comment by gravino81** [comment]
+1

**Comment by xbuzz** [comment]
+1

**Comment by kkankala** [comment]
+1

**Comment by rhyswynne** [comment]
+1

**Comment by topfalade** [comment]
+1

**Comment by anshbansal** [comment]
+1

**Comment by hoyangtsai** [comment]
+1

**Comment by legosalogos** [comment]
+1

**Comment by teesche** [comment]
+1

**Comment by fergalmoran** [comment]
Struggling to see why 4 years on this still hasn't been implemented.
@MichaelTunnell - the use case is simple. I want 2 (or more) buttons on my site, one is for the standard RSS feed (using the standard RSS icon), another is for a "Subscribe in Feedly" button, for which I would like to use a Feedly icon.

**Comment by MichaelTunnell** [comment]
I understand but with the new fontawesome you can have a lot of variety based on the normal, light and bold versions of the icons.

> Struggling to see why 4 years on this still hasn't been implemented.

This is a brand. They can just take and use a brand's logo without their permission so it isn't a simple task regardless of the desire of the community.

**Comment by sanek4life** [comment]
+1

**Comment by svetanaumoff** [comment]
+1

Omg, it's been 10 years! Please, please, add Feedly icon. Plus, Bloglovin is dead now, and lots of its users are now on Feedly. General RSS icon is not an option. It's an addition, a variety. Thank you!

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
